src/library_webgl.js

```diff
@@ -2365,12 +2368,12 @@ var LibraryGL = {
 #if GL_ASSERTIONS
     assert(GL.currentContext.version >= 2);
 #endif
-    GLctx.uniform1iv(webglGetUniformLocation(location), HEAP32, value>>2, count);
+    count && GLctx.uniform1iv(webglGetUniformLocation(location), HEAP32, value>>2, count);
```

Comment:
Is this just another way of writing `if (count) ...`. but saving two bytes?

Seems a little odd to do the micro-optimization here but not globally across the codebase.

Reply:
> Is this just another way of writing if (count) .... but saving two bytes?

That's right.

> Seems a little odd to do the micro-optimization here but not globally across the codebase.

Maybe. It is used consistently in WebGL library, and that is probably more on the hot code size path that some of the other libraries.

Reply:
Ah, ok. I didn't realize there was precedent for this.

Reply:
I think closure does this? If not, it would be easy to add a js optimizer pass. I think depending on the optimizer for this kind of thing is probably best (keeps code more readable and the optimization applies to more places).